package.json

```diff
@@ -20,9 +20,9 @@
     "@changesets/cli": "^2.27.8",
     "@srcbook/configs": "workspace:^",
     "@types/node": "^20.14.2",
+    "eslint": "^8.57.0",
```

Comment:
Not a big deal but why change this order?

Reply:
Ahh, we can ignore this during the integration I was getting a type error which I thought could be resolved by having ai-sdk for Google installed in the root package when I later realized it has nothing to with it, I removed it from package.json for avoiding not so necessary installation in the package.

Reply:
And honestly I didn't realized that it changed the order.

Reply:
We don't need this particular commit as well btw! I don't know how it got committed because I remembered I removed the dependency, if it's possible to not align with the following package.json then it works because we have nothing to do with root's package.json file